openfaas/faas
# Bump nats-streaming version to 0.11.0

## Description
- Bump the nats-streaming version to 0.11.0 in the docker-compose files, this is the latest tag published on Docker Hub
- Addresses #790

## Motivation and Context




## How Has This Been Tested?
I have started by testing this locally with 

1. create a request callback bin here https://webhook.site and set the bin as a variable 
```sh
export hook=<>
```
2. deploy using the `./deploy-stack.sh`
3. deploy the default stack in the faas repo using `faas-cli deploy`
4. testing the functions using sync and async calls, e.g.
```sh
$ echo "$SECONDS" | faas-cli invoke -a --header "X-Callback-Url=$hook" stronghash
Function submitted asynchronously.
$ echo "$SECONDS" | faas-cli invoke -a --header "X-Callback-Url=$hook" ruby-echo
Function submitted asynchronously.
```
5. watch the queue worker logs to see it respond
```sh
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | [#10] Received on [faas-request]: 'sequence:10 subject:"faas-request" data:"{\"Header\":{\"Accept-Encoding\":[\"gzip\"],\"Content-Length\":[\"5\"],\"Content-Type\":[\"text/plain\"],\"User-Agent\":[\"Go-http-client/1.1\"],\"X-Call-Id\":[\"e955a278-f77d-4d9d-a558-8208c3528f2d\"],\"X-Callback-Url\":[\"https://webhook.site/3f2bd1c9-bf28-4a5c-bc46-f26b161c80ca\"],\"X-Start-Time\":[\"1538579432034861700\"]},\"Host\":\"127.0.0.1:8080\",\"Body\":\"NDI2MQo=\",\"Method\":\"POST\",\"Path\":\"\",\"QueryString\":\"\",\"Function\":\"stronghash\",\"CallbackUrl\":{\"Scheme\":\"https\",\"Opaque\":\"\",\"User\":null,\"Host\":\"webhook.site\",\"Path\":\"/3f2bd1c9-bf28-4a5c-bc46-f26b161c80ca\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"}}" timestamp:1538579432035416400 '
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Request for stronghash.
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Callback to: https://webhook.site/3f2bd1c9-bf28-4a5c-bc46-f26b161c80ca
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Wrote 132 Bytes
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | 200 OK
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Posted result: 200
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Posting report - 200
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Request for ruby-echo.
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | [#11] Received on [faas-request]: 'sequence:11 subject:"faas-request" data:"{\"Header\":{\"Accept-Encoding\":[\"gzip\"],\"Content-Length\":[\"5\"],\"Content-Type\":[\"text/plain\"],\"User-Agent\":[\"Go-http-client/1.1\"],\"X-Call-Id\":[\"9d65755a-46c6-4a4f-b5fc-ba0317c788c0\"],\"X-Callback-Url\":[\"https://webhook.site/3f2bd1c9-bf28-4a5c-bc46-f26b161c80ca\"],\"X-Start-Time\":[\"1538579697168874900\"]},\"Host\":\"127.0.0.1:8080\",\"Body\":\"NDUyNgo=\",\"Method\":\"POST\",\"Path\":\"\",\"QueryString\":\"\",\"Function\":\"ruby-echo\",\"CallbackUrl\":{\"Scheme\":\"https\",\"Opaque\":\"\",\"User\":null,\"Host\":\"webhook.site\",\"Path\":\"/3f2bd1c9-bf28-4a5c-bc46-f26b161c80ca\",\"RawPath\":\"\",\"ForceQuery\":false,\"RawQuery\":\"\",\"Fragment\":\"\"}}" timestamp:1538579697169399500 '
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Wrote 43 Bytes
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | 200 OK
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Callback to: https://webhook.site/3f2bd1c9-bf28-4a5c-bc46-f26b161c80ca
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Posted result: 200
func_queue-worker.1.t3nelzdypshs@linuxkit-025000000001    | Posting report - 200
```
6. check the webhook bin to see the function output 
![image](https://user-images.githubusercontent.com/891889/46420355-2650fe80-c730-11e8-9fce-93cc777f318d.png)

So far, it seems to work as expected.

I also tested monitoring nats-streaming locally with the `nats-top` tool

## Types of changes


## Checklist:
